backend/src/controllers/quiz.ts

```diff
@@ -12,9 +12,9 @@ const createQuiz: RequestHandler = async (req, res, next) => {
     const category = req.body.category;
     const questionList = req.body.questionList;
     const answers = req.body.answers;
-    const passing_percentage=req.body.passing_percentage;
-    
-    const quiz = new Quiz({ name, questionList, answers, passing_percentage, createdBy });
+    const passing_percentage = req.body.passing_percentage;
```

Comment:
change everything to camelCase.

Reply:
Okay Sir

Reply:
passing_percentage should be passingPercentage

Reply:
Got it Sir